_posts/2022-12-21-easily-verify-your-rust-in-ci-with-kani.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "Easily verify your Rust in CI with Kani and Github Actions"
+---
+
+Formal verification tools, such as Kani, are valuable because they provide a way to definitively check, using mathematical techniques, whether a property of your code is true under all circumstances.
```

Comment:
I wonder if the first sentence would be punchier if broken up, there's a lot to unpack here at the moment.

I also think that there is an implicit leap from "property of your code" to "catching bugs". It might be obvious to us that the 'property' here is bug-freedom of some sort, but I think this is jargon. Can I suggest:
```suggestion
Formal verification tools like Kani definitively check that certain classes of bugs will not occur in your program under any circumstance. Kani uses mathematical techniques to explore your code over all inputs, meaning that while testing is a great way to catch the *first* bug, formally verifying your code with Kani guarantees that you've caught the *last* bug.
```